app/views/recipes/new.html.haml

```diff
@@ -8,7 +8,7 @@
   = form.fields_for :yeasts do |yeast_fields|
     %p
       = yeast_fields.text_field :name, placeholder: 'Yeast name'
-      = yeast_fields.number_field :attenuation, placeholder: 'Yeast attenuation', min: 0.5
+      = yeast_fields.number_field :attenuation, placeholder: 'Yeast attenuation'
```

Comment:
Yeah, I should have done that ages ago. I'll do that once I merge this into master so that the form has the bootstrap classes.